### ISSUE TITLE
rolling and main branches are not in sync

### ISSUE THREAD

**Issue by esteve** [autowarefoundation/autoware_msgs#93]
### Checklist

- [X] I've read the [contribution guidelines](https://github.com/autowarefoundation/autoware/blob/main/CONTRIBUTING.md).
- [X] I've searched other issues and no duplicate issues were found.
- [X] I'm convinced that this is not my fault but a bug.

### Description

Same as https://github.com/autowarefoundation/autoware_adapi_msgs/issues/56

### Expected behavior

Same as https://github.com/autowarefoundation/autoware_adapi_msgs/issues/56

### Actual behavior

Same as https://github.com/autowarefoundation/autoware_adapi_msgs/issues/56

### Steps to reproduce

Same as https://github.com/autowarefoundation/autoware_adapi_msgs/issues/56

**Comment by esteve** [comment]
@youtalk given that `main` and `rolling` haven't diverged, we can fix this by renaming `rolling` to `main` and removing `main`. Alternatively, we could pull --ff-only from `rolling` to `main` (NOT MERGE) to preserve the same commit ids and then push to `main`. If we open a PR from `rolling` to `main` it might not work, we'd need to push to `main` directly.

**Comment by esteve** [comment]
Also, in order for https://github.com/autowarefoundation/autoware_msgs/issues/93#issuecomment-2160955700 to work, we can't merge any PRs until this is resolved so that the commit history is kept intact.

**Comment by youtalk** [comment]
@esteve Yes, indeed. We should change the branch to main.

**Comment by esteve** [comment]
@xmfcx @mitsudome-r can I have admin access to this repo? Or at least, have write access to main? This is what I want to do:

```shell
esteve@haggis:/tmp$ git clone git@github.com:autowarefoundation/autoware_msgs.git
Cloning into 'autoware_msgs'...
remote: Enumerating objects: 617, done.
remote: Counting objects: 100% (362/362), done.
remote: Compressing objects: 100% (224/224), done.
remote: Total 617 (delta 206), reused 227 (delta 133), pack-reused 255
Receiving objects: 100% (617/617), 243.76 KiB | 834.00 KiB/s, done.
Resolving deltas: 100% (311/311), done.
esteve@haggis:/tmp$ cd autoware_msgs/
esteve@haggis:/tmp/autoware_msgs$ git pull --ff-only origin rolling
From github.com:autowarefoundation/autoware_msgs
 * branch            rolling    -> FETCH_HEAD
Updating 8e657ab..4c3a193
Fast-forward
 autoware_common_msgs/CHANGELOG.rst       | 16 ++++++++++++++++
 autoware_common_msgs/package.xml         |  2 +-
 autoware_control_msgs/CHANGELOG.rst      | 10 ++++++++++
 autoware_control_msgs/package.xml        |  2 +-
 autoware_localization_msgs/CHANGELOG.rst | 10 ++++++++++
 autoware_localization_msgs/package.xml   |  2 +-
 autoware_map_msgs/CHANGELOG.rst          | 34 ++++++++++++++++++++++++++++++++++
 autoware_map_msgs/package.xml            |  2 +-
 autoware_perception_msgs/CHANGELOG.rst   | 49 +++++++++++++++++++++++++++++++++++++++++++++++++
 autoware_perception_msgs/package.xml     |  2 +-
 autoware_planning_msgs/CHANGELOG.rst     | 61 +++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++
 autoware_planning_msgs/package.xml       |  2 +-
 autoware_sensing_msgs/CHANGELOG.rst      | 11 +++++++++++
 autoware_sensing_msgs/package.xml        |  2 +-
 autoware_system_msgs/CHANGELOG.rst       | 17 +++++++++++++++++
 autoware_system_msgs/package.xml         |  2 +-
 autoware_vehicle_msgs/CHANGELOG.rst      | 38 ++++++++++++++++++++++++++++++++++++++
 autoware_vehicle_msgs/package.xml        |  2 +-
 18 files changed, 255 insertions(+), 9 deletions(-)
 create mode 100644 autoware_common_msgs/CHANGELOG.rst
 create mode 100644 autoware_control_msgs/CHANGELOG.rst
 create mode 100644 autoware_localization_msgs/CHANGELOG.rst
 create mode 100644 autoware_map_msgs/CHANGELOG.rst
 create mode 100644 autoware_perception_msgs/CHANGELOG.rst
 create mode 100644 autoware_planning_msgs/CHANGELOG.rst
 create mode 100644 autoware_sensing_msgs/CHANGELOG.rst
 create mode 100644 autoware_system_msgs/CHANGELOG.rst
 create mode 100644 autoware_vehicle_msgs/CHANGELOG.rst
esteve@haggis:/tmp/autoware_msgs$ git push
ERROR: Permission to autowarefoundation/autoware_msgs.git denied to esteve.
fatal: Could not read from remote repository.

Please make sure you have the correct access rights
and the repository exists.
```

Or if you want to do it yourselves, the `--ff-only` flag for `git pull` is very important so that `rolling` is not merged into `main`, the commits from `rolling` need to be in the linear history. Thanks.

**Comment by esteve** [comment]
Fixed in 4c3a1939520cd8ac9d05b0addf7286a0534de024, thanks @youtalk @xmfcx @mitsudome-r

**Comment by esteve** [comment]
I've submitted https://github.com/ros/rosdistro/pull/41606 to update the branch in the `rosdep` database.
